.cargo/config.toml

```diff
@@ -0,0 +1,8 @@
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "target-feature=+bulk-memory"]
+
+[target.i686-pc-windows-msvc]
```

Comment:
1. Both Windows cases can be unified to a single case.
2. We should consider enabling static CRT only in release mode, as it might hurt debug build times (Haven't tested it yet though):
```toml
[target.'cfg(all(target_os = "windows", not(debug_assertions)))']
rustflags = ["-C", "target-feature=+crt-static"]
```
3. Please add a trailing newline at end-of-file :)

Reply:
It can be even simpler: `[target.'cfg(all(windows, not(debug_assertions)))']`